### PR TITLE
add missing peer deps, and fix cra template scss 

### PR DESCRIPTION
**Pull request by aruniverse**
Fix scss so theres no gap below the viewer and the app takes up the entire screen.
Add missing peer deps required by the introduction of the new tree-widget-react and property-grid react in the viewer.
Bump property-grid-react

### CODE REVIEW

**Review by kckst8 — COMMENTED** [review]
Why do we need to bump the min imjs version?

**Comment by aruniverse** [comment]
> Why do we need to bump the min imjs version?

Because property-grid-react has a peer dep on ^2.9.2. Need to determine if that can be lowered or not on that pkg's end

**Comment by kckst8** [comment]
> > Why do we need to bump the min imjs version?
> 
> Because property-grid-react has a peer dep on ^2.9.2. Need to determine if that can be lowered or not on that pkg's end

I would prefer we do that if possible. I haven't noticed any issues with the property grid running the sample app on 2.6.4. If we do need to bump, we should bump the min in the itv package as well

**Comment by aruniverse** [comment]
> > > Why do we need to bump the min imjs version?
> > 
> > 
> > Because property-grid-react has a peer dep on ^2.9.2. Need to determine if that can be lowered or not on that pkg's end
> 
> I would prefer we do that if possible. I haven't noticed any issues with the property grid running the sample app on 2.6.4. If we do need to bump, we should bump the min in the itv package as well

Updated property-grid-react to have 2.6.0 as min supported version

**Review by kckst8 — CHANGES_REQUESTED** [review]

File: packages/modules/itwin-viewer-react/package.json [hunk 1]
```diff
@@ -100,6 +100,7 @@
   },
   "peerDependencies": {
     "@bentley/bentleyjs-core": "^2.6.4",
+    "@bentley/context-registry-client": "^2.6.4",
```
[the remark this answers is not in the document]
Comment (aruniverse) [reply packages/modules/itwin-viewer-react/package.json:103]: Its a dev+peer dep of  [tree-widget-react](https://www.npmjs.com/package/@bentley/tree-widget-react), but its a direct dep of a few core packages, https://www.npmjs.com/package/@bentley/context-registry-client, so it should already be getting pulled in i think.

**Pull request closed without merging**